agent/stats/engine_unix_test.go

```diff
@@ -207,7 +213,13 @@ func TestFetchEBSVolumeMetrics(t *testing.T) {
 	engine.cluster = defaultCluster
 	engine.containerInstanceArn = defaultContainerInstance
 	engine.client = mockDockerClient
-	engine.csiClient = csiclient.NewDummyCSIClient()
+
+	mockCsiClient := mock_csiclient.NewMockCSIClient(mockCtrl)
+	mockCsiClient.EXPECT().GetVolumeMetrics(gomock.Any(), "vol-12345", "/mnt/ecs/ebs/taskarn_vol-12345").Return(&csiclient.Metrics{
+		Used:     15 * 1024 * 1024 * 1024,
+		Capacity: 20 * 1024 * 1024 * 1024,
+	}, nil).MinTimes(1)
```

Comment:
Is the exact number of `GetVolumeMetrics` calls deterministic? We may want to use `Times(<that number>)` if so to make the assertion stricter.

Reply:
oh yeah that's a better idea, thanks!